interfaces/cython/cantera/test/test_transport.py

```diff
@@ -88,6 +88,17 @@ def test_add_species_multi(self):
         self.assertNear(gas1.thermal_conductivity, gas2.thermal_conductivity)
         self.assertArrayNear(gas1.multi_diff_coeffs, gas2.multi_diff_coeffs)
 
+    def test_species_visosities(self):
+
```

Comment:
Blank line is unnecessary (minor nitpick)

Reply:
Removed